android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/BaseRegisterActivity.kt

```diff
@@ -81,7 +81,7 @@ abstract class BaseRegisterActivity :
 
   private lateinit var registerViewModel: RegisterViewModel
 
-  private lateinit var registerActivityBinding: BaseRegisterActivityBinding
+  protected lateinit var registerActivityBinding: BaseRegisterActivityBinding
```

Comment:
We do not need to make this `protected` populate the `configurableViews` map and access them in the subclass if you need to extend any functionalities provided via the base register